getting-started/quick-start.md

```diff
@@ -0,0 +1,110 @@
+# Getting Started with Amber
+This getting started guide will help you get a full stack web application running in just a few minutes by leveraging amber’s code generators.
+
+After we have a working application, then we will walk through the code that is making it all happen and code some enhancements along the way.
+
+Let’s get started!
+
+## Install dependencies
+If you already have `crystal`, `amber`, and a database installed, you can skip this step.
+
+### Install `crystal`
+
+#### OS X with homebrew
+ ```
+$ brew install crystal-lang
+```
+#### Ubuntu or Debian
+ ```
+$ curl https://dist.crystal-lang.org/apt/setup.sh | sudo bash
+$ sudo apt-get install build-essential crystal
+```
+
+#### Full Install Instructions
+See full installation instructions [here](https://crystal-lang.org/docs/installation/)
+
+### Install `amber`
+
+#### OS X with homebrew
+```
+brew install amberframework/amber/amber
+```
+
+#### Ubuntu or Debian
```

Comment:
This should be labeled:

#### Installing from Source

Since it's not uniq to ubuntu or debian, Although I guess the deps are.